tools/pylib/zoidberg/field.py

```diff
@@ -1595,7 +1615,16 @@ def field_values(r, phi, z, configuration=0, plot_poincare=False):
                 pos.x3 = x3[slc]
 
                 ## Call tracer service
-                res = tracer.service.magneticField(pos, config)
+                redo = 2
+                while redo:
+                    try:
+                        res = tracer.service.magneticField(pos, config)
+                    except:
+                        redo -= 1
```

Comment:
Putting a short `sleep` here might be wise, if this is to help with intermittent network failures